TestDiffCompatibility is flaky

**TestDiffCompatibility** has been detected as a flaky test.

- **Package:** ``
- **Category:** test_flake
- **Occurrences:** 27498
- **First seen:** 2026-07-10
- **Last seen:** 2026-07-14

**CI Run:** https://github.com/pulumi/pulumi/actions/runs/29345297608

**Test output:**
```
Value***V:""***, "⟪A₭˧":resource.PropertyValue***V:(*asset.Asset)(0xc00100c730)***, "�%":resource.PropertyValue***V:interface ***(nil)***, "(〇":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:true***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "~ɎȺA\u2061a~^\u009f꜔A\x1b𞱹ῼ", "\u00a0؆ᛯ", "¤\x00", "֎^Aaa#𞥋\u00a0%̂ႃA+", "؇\x01¦*", "𞥃A"***, "a𠒱\u0605Ű˫.⸂A*`�⅜aa?ǅA̮aa٘𞥋?+㑚҈":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:-0.396616440249285***, resource.PropertyValue***V:false***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:(*archive.Archive)(0xc00100ca00)***, resource.PropertyValue***V:"\u2029˄Ⱥ^"***, resource.PropertyValue***V:(*asset.Asset)(0xc00100ca50)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:(*asset.Asset)(0xc00100caa0)***, resource.PropertyValue***V:-1.6279203954232225e+14***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"́᪙"***, resource.PropertyValue***V:true***, "Ⱥ\";᠄#\u200a$𒑤a\U00105467/ᶻ?\r\ue39a!":resource.PropertyValue***V:-2.717373668856481e-235***, "פ":resource.PropertyValue***V:[]resource.PropertyValue***, "۩":resource.PropertyValue***V:[]resource.PropertyValue***, "᳗+̃*A":resource.PropertyValue***V:resource.PropertyMap***"̠!_=":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "ᾉAA\ue007aॊ\"A᧤a":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*asset.Asset)(0xc00100c640)***, resource.PropertyValue***V:interface ***(nil)***, "\u2001ᾛ\u0604ᶿ=דˣ":resource.PropertyValue***V:"󠆼 -"***, "\u2002ෝ":resource.PropertyValue***V:(*resource.Secret)(0xc0008243d0)***, "\u2008C":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*archive.Archive)(0xc00100c7d0)***, resource.PropertyValue***V:"/\ue001ȺK+\tǂ\u00a0"***, resource.PropertyValue***V:false***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"➟בa¸৺\ue011A¢"***, resource.PropertyValue***V:"ꛨ"***, resource.PropertyValue***V:(*asset.Asset)(0xc00100c820)***, resource.PropertyValue***V:(*asset.Asset)(0xc00100c870)***, resource.PropertyValue***V:"³^!𑃹"***, resource.PropertyValue***V:true***, resource.PropertyValue***V:1.1661010420456013e-32***, resource.PropertyValue***V:4.2935779625e+07***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"𧛅\"ែa౼!"***, resource.PropertyValue***V:1.7976931348623157e+308***, resource.PropertyValue***V:"#ᾍa"***, resource.PropertyValue***V:"ि\ufeff\ve︡\x00䷡؇"***, resource.PropertyValue***V:true***, resource.PropertyValue***V:false***, resource.PropertyValue***V:true***, resource.PropertyValue***V:(*asset.Asset)(0xc00100c8c0)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:true***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"@~"***, resource.PropertyValue***V:(*asset.Asset)(0xc00100c910)***, "\ue28aA฿\x7fa\x00ਿȺى\ufeff[\u0603\r/=*=a䁟AA(\u008e৴.=\x06\u2004Ⱥ??\x1b":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "?|\x00\U000e0078"***, "﹩Ą":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:""***, resource.PropertyValue***V:false***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:"ো¢₸Ⱥෙ~a:'"***, "𖾙Ⱥ[a#":resource.PropertyValue***V:(*resource.Secret)(0xc000824580)***, resource.PropertyValue***V:(*resource.Secret)(0xc000824670)***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"꙲\U0001343d~aǰ౺~ƻ"***, resource.PropertyValue***V:false***, resource.PropertyValue***V:false***, resource.PropertyValue***V:(*asset.Asset)(0xc00100cc80)***, resource.PropertyValue***V:(*archive.Archive)(0xc00100ccd0)***, resource.PropertyValue***V:-0.274256029303829***, resource.PropertyValue***V:false***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:""***, resource.PropertyValue***V:interface ***(nil)***, "!_*\u00a0\x1b":resource.PropertyValue***V:(*asset.Asset)(0xc00100d090)***, "\"ऻ`ª@T(":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*archive.Archive)(0xc00100cb90)***, resource.PropertyValue***V:""***, resource.PropertyValue***V:"\v"***, resource.PropertyValue***V:(*archive.Archive)(0xc00100cbe0)***, resource.PropertyValue***V:-6.976990631061371e+09***, resource.PropertyValue***V:(*asset.Asset)(0xc00100cc30)***, "(A\u2009A*\U00013431ʱ\x00=\u202e*":resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:false***, "\tA-ऻ":resource.PropertyValue***V:(*archive.Archive)(0xc00100ce10)***, " ף⃝\v":resource.PropertyValue***V:false***, "#º":resource.PropertyValue***V:"?"***, "1c@@\taཻA🇼~?":resource.PropertyValue***V:-7.6082071210638215e-121***, "?A":resource.PropertyValue***V:(*archive.Archive)(0xc00100cf50)***, "~\u0603~~/~-𑶖_ⅶ":resource.PropertyValue***V:-1.994354248046875***, "~ᾼa":resource.PropertyValue***V:"६Ꭰ^A _!⩀"***, "Ⱥ\U000110cdʲ ҈\x7f\U000fa0b8۶˂":resource.PropertyValue***V:(*archive.Archive)(0xc00100cd70)***, "\u08e2_@=$$":resource.PropertyValue***V:"©ꛪȺ🯹\ŕa\t℔/?𞲰\ue000Ⅺ¼ꛩ౼ō\U000fb938~/̅[\x7fAヵ"***, "A\x1bρ$\"\u00a0~ұ⃣A|A.ʰ\U000e005a⳦\x00€=⤾⃝":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:""***, resource.PropertyValue***V:"ᛯ.৳۾ʰ𑁧"***, resource.PropertyValue***V:"?@%\x05\u202e$²𝪞์#൶ꜵ[ˡʰ"***, resource.PropertyValue***V:false***, resource.PropertyValue***V:true***, resource.PropertyValue***V:"\\a"***, resource.PropertyValue***V:""***, resource.PropertyValue***V:"⃞Ⱥ\rिA\U000e007da\U00103b98"***, resource.PropertyValue***V:2.9917663430310153e+51***, "́<'\"Ⱥ/\U000f5561ৈ!\x1a\U0010ca34ႄ\U000e0001":resource.PropertyValue***V:resource.PropertyMap***, "҂(\u0601©":resource.PropertyValue***V:(*resource.Secret)(0xc0008246e0)***, "ණ󠇯":resource.PropertyValue***V:false***, "𞥟༱Ⱥ":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:false***, resource.PropertyValue***V:true***, resource.PropertyValue***V:(*archive.Archive)(0xc00100cfa0)***, resource.PropertyValue***V:-6.931597066239075e-191***, resource.PropertyValue***V:false***, resource.PropertyValue***V:"-"***, resource.PropertyValue***V:true***, resource.PropertyValue***V:(*archive.Archive)(0xc00100cff0)***, resource.PropertyValue***V:"?=Ⱥ,\ue00e⺬\""***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:"\U000e002d<fA&⃜ᛯ[?~\"\u205f$#"***, resource.PropertyValue***V:"\r\u0087:?;𐅄\vȺ"***, resource.PropertyValue***V:(*asset.Asset)(0xc00100d040)***, resource.PropertyValue***V:(*resource.Secret)(0xc0008248c0)***, "ᛯ◸a፻ᾘ|˃\rᵡ˕[\v´\"~-ऻ\ra\x1bᛰȺ~~,a":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:":ः@"***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "\x00", " :¢$ˋ", "1=|aª?(af@ῼ?௶\u3000", "^", "\x7fˊȺ", "ѩ῏𞲰", "؎⃫ǀ", "੩!ᆝᵌ;~ؿD𒐫~ᾎ\u2002˪\ue02e"***, "ᾈ":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:true, Dependencies:[]urn.URN***"", "$\u0605", "A", "A@ↆ***ʱ⠲\a<﮻סⅢ൰", "Ⱥ", "৹<", "𞴵?𝋉🏻👤\x00᥇", "󠇯?aᛮ"***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*resource.Secret)(0xc000825bc0)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:"ǃʰ״,~~@\ue126&"***, Known:true, Secret:false, Dependencies:[]urn.URN***"["***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:false***, "$\u00a0":resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:-48.92678650607653***, resource.PropertyValue***V:(*resource.Secret)(0xc000825c20)***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:"`_\"\ue00c\u2006ȺZ߾ \x1b\t'&ǋ?₰"***, "a߿ʹៗ#a\ue00e🏼a=A":resource.PropertyValue***V:true***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:true, Dependencies:[]urn.URN***"", "#༨?\x7f`<aaȺd\ufeff~_װ"***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:(*asset.Asset)(0xc00110fb80)***, Known:true, Secret:true, Dependencies:[]urn.URN***"", "<", "h", "ǋ", "﮳⽊A¸\t?"***, Known:true, Secret:false, Dependencies:[]urn.URN***"", " $?\u00adAA\v", "?$$+ᛮ𒑃\x00¨;ᶯ", "[D\ue002", "~✎a+&A\u2000a", "\u0604༭", "\u2007-Ư🫅aȺ'᰽?:A\ue000˂ᴬ\ufffb", "𑅱~?", "𣬞-+4⃞𑥖", "\U000e0075 a;&#"***, "♮=;$":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:true, Dependencies:[]urn.URN(nil)***, "꙰=Ű̅?":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:false, Dependencies:[]urn.URN***"", "\t৷\t 0ג", "\v₼#?५⅁", "*.⃠'!ᛯa\u00ad~a?+4", "@a\u0600a", "ᾬȺː\u0602ᵜ́/*\ue1e2႒ב𐩓ʰ㏊\"\\ᛯ\x7fᾩʲ\x7fªȺa%a"***, resource.PropertyValue***V:resource.PropertyMap***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:(*asset.Asset)(0xc00110f7c0)***, "\x00":resource.PropertyValue***V:(*archive.Archive)(0xc00110f770)***, "&?b0,ᡴ":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:[]resource.PropertyValue***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "\x1b?󠇯֍", " ?໒\x00Ồ︗@؆B", "ȺCª˻", "Ⱥaa@:&Ċ_)***$𑶔Ⅰǅ\u2029©찕", "Ⱥ\x7f\u0602?𝒍\u1680d", "Ѯ$^ڗa&", "҈", "﮽r_[/", "\U000e007f£(̂❿"***, "=?꜇":resource.PropertyValue***V:resource.PropertyMap***"�A":resource.PropertyValue***V:[]resource.PropertyValue***, "^º³ǅ":resource.PropertyValue***V:resource.PropertyMap***"a<�˸b🄃!D∶":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:(*asset.Asset)(0xc00110f720)***, resource.PropertyValue***V:""***, "⩳ᾟ𝋉":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:false, Dependencies:[]urn.URN***"a𑇇\u2006A\u0604`𞁐", "~a`b<̖\r₅𑧜@"***, "aA":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:true, Secret:true, Dependencies:[]urn.URN***"", "/⃝\ue001&\u2006\r<;", "൵�~"***, "ac፬\ue001":resource.PropertyValue***V:(*archive.Archive)(0xc00110f6d0)***, "~$\u0601$$":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:false***, Known:true, Secret:true, Dependencies:[]urn.URN***"", "*˝〥2", "@#", "`T𑇎J\U000fbaf2@\U000e007fջ3+R*꭫", "|௲a\ue012aᤵ⅄", "ǈ", "Ⱥ#ᾞǈ̆0", "ः", "ী\t@\u2001~A(\\\x01³\r\u202eȺ﹀a$A\x05º", "꙲̟#a⃞?=£"***, resource.PropertyValue***V:(*resource.Secret)(0xc000825a10)***, resource.PropertyValue***V:2.5363252651227747e-222***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*asset.Asset)(0xc00110f9f0)***, resource.PropertyValue***V:resource.PropertyMap***"̅":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:true***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "༇҈"***, "ⲻ&aᛮ︬\u0600.؏؉%^$aa&":resource.PropertyValue***V:false***, resource.PropertyValue***V:(*resource.Secret)(0xc000825a50)***, resource.PropertyValue***V:(*resource.Secret)(0xc000825a60)***, resource.PropertyValue***V:1909.126953125***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:""***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "'a*꣒a͗", "=©+Ⱥ<⇹*́\x00꙱∣-\u2002Aa"***, resource.PropertyValue***V:true***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:[]resource.PropertyValue***, Known:true, Secret:true, Dependencies:[]urn.URN***"", "a~", "°ˇ"***, "\ue026":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:false, Dependencies:[]urn.URN***"|ᩮ\t/'"***, "\ue153֏***a":resource.PropertyValue***V:(*asset.Asset)(0xc00110fb30)***, "\U000f048c.":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:"꤇G"***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:interface ***(nil)***, "#\x14∕":resource.PropertyValue***V:[]resource.PropertyValue***, "2A":resource.PropertyValue***V:resource.PropertyMap***"៛#\u0600\v*¢ᾉ":resource.PropertyValue***V:interface ***(nil)***, "=້ଂ-":resource.PropertyValue***V:"[\v"***, "@+a( ₿":resource.PropertyValue***V:[]resource.PropertyValue***, "AA\U001020d7²<ũⅴ$Ⅰ~¤":resource.PropertyValue***V:false***, "AᛮⅠ":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:[]resource.PropertyValue***, resource.PropertyValue***V:resource.PropertyMap***"#Aો=𐞎̄\U000f5d34ȫ":resource.PropertyValue***V:""***, "ঃ+𐭘🯊":resource.PropertyValue***V:(*asset.Asset)(0xc00100d400)***, "୲₷𞻱~***ۖ":resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:false, Dependencies:[]urn.URN***"⸣\u2000"***, resource.PropertyValue***V:resource.PropertyMap***"$%[~`\x11!$\x1d\ufeff̴$":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "%a\u202ea\x00(a~(\u2001?¦ꛮ":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "(%^؆\"":resource.PropertyValue***V:(*asset.Asset)(0xc00100d450)***, "<\ufeff$B²[":resource.PropertyValue***V:interface ***(nil)***, "A":resource.PropertyValue***V:interface ***(nil)***, "A@\\A\u0090":resource.PropertyValue***V:"\x7f"***, "a꙱%²+\vA":resource.PropertyValue***V:"A"***, "¯\u202c\u0090₷":resource.PropertyValue***V:false***, "\u202eਾa\x01\x03(A\u200b\"¯/?ª\ufeff߾b𒑲!𞤡৹;̀":resource.PropertyValue***V:3.679049400808561e-11***, "\u202f⎔*𑙐₽?":resource.PropertyValue***V:(*asset.Asset)(0xc00100d4f0)***, "┘ⅸڟ   ":resource.PropertyValue***V:(*asset.Asset)(0xc00100d4a0)***, resource.PropertyValue***V:(*asset.Asset)(0xc00100d540)***, resource.PropertyValue***V:"A𞥟a~⃝~¨𝟉҉\ue000𑓗\\+䆂᎕Ⅾࠜ\x00^\U000f45ce󠅲"***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:(*asset.Asset)(0xc00100d590)***, Known:true, Secret:false, Dependencies:[]urn.URN***"\x1b^"***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:"#z?"***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "۾⁞\x00^~", "᤺", "ⅵ𞻱\"⃣a"***, resource.PropertyValue***V:(*asset.Asset)(0xc00100d5e0)***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "\x00F":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "!":resource.PropertyValue***V:9.948936528076042e-117***, "#\u0600##$":resource.PropertyValue***V:(*asset.Asset)(0xc00100d8b0)***, "%":resource.PropertyValue***V:"\u3000!ᥚ"***, "%-²𝆹𝅥𝅮@𞁄\x7f<𑈵\U000e0023aǲ\U00013439I´¯�⃠\\":resource.PropertyValue***V:"|^³\b#-%~ǅ<\x06ऻ𞲰***𑤾\ue01cȽ؇୵a"***, "%^":resource.PropertyValue***V:(*archive.Archive)(0xc00100d810)***, "5":resource.PropertyValue***V:interface ***(nil)***, "?\x00a෯¢؆ʱ":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "?ª\\":resource.PropertyValue***V:false***, "A𑜦\x005Ⱥ":resource.PropertyValue***V:"-,A©ⅳ"***, "[-\x01𝪩:":resource.PropertyValue***V:"_,𝜵#?~\x00<"***, "_:\x15:*₠\"":resource.PropertyValue***V:"%𔔶a˔"***, "a":resource.PropertyValue***V:1.205438275906829***, "a>©-ª\v⃢A":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "a^̀A!\t":resource.PropertyValue***V:-3.6796790066281188e+236***, "a\x7f":resource.PropertyValue***V:(*archive.Archive)(0xc00100d900)***, "a˝\"̭Ⱥ$\u202e":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "\u00ad\";A":resource.PropertyValue***V:1.79340420567611e-134***, "ʱ+":resource.PropertyValue***V:6.734887500725754e-125***, "̚إa\"a𐅟=":resource.PropertyValue***V:(*archive.Archive)(0xc00100d860)***, "ͭ⊋̃a$⃤Ⱥ\x1b~ᾩ":resource.PropertyValue***V:0.7338564373296077***, ";,؎ाaa":resource.PropertyValue***V:"\x05#"***, "؆***":resource.PropertyValue***V:(*asset.Asset)(0xc00100d630)***, "࡙֍Aʱ#":resource.PropertyValue***V:"ⴃ𜽅\x00\ue000\u2007\u061c;"***, "୲ᴙἥAA\x00˅䖇\u202e½.℥Ⅰ\u200b+\rȺ|⎝Œಃ🗙a⃝#":resource.PropertyValue***V:"A%"***, "ᛮ":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "᩺AᾟC҈🗸~'\n|Ṁ؎aᾩᾏ𖺘":resource.PropertyValue***V:""***, "⃢":resource.PropertyValue***V:0.004506172624004989***, resource.PropertyValue***V:true***, resource.PropertyValue***V:(*resource.Secret)(0xc000824f60)***, resource.PropertyValue***V:[]resource.PropertyValue***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*archive.Archive)(0xc00100d950)***, resource.PropertyValue***V:false***, resource.PropertyValue***V:true***, resource.PropertyValue***V:-2.4160495497277344e+10***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:1.27258738933996e+23***, resource.PropertyValue***V:(*resource.Secret)(0xc000824fb0)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:(*archive.Archive)(0xc00100d9a0)***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "$ƻ፭a$", "%A", "a\u2001$\"a\"@൘<\ue00dყ a𝂧", "𐅦[***~", "𑼾;~"***, resource.PropertyValue***V:[]resource.PropertyValue***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:(*archive.Archive)(0xc00100d9f0)***, Known:true, Secret:false, Dependencies:[]urn.URN***"", " \"\x7f\ue000", "(҈ʰ? Ⱥ߇≯´́﹎ᛮ\x00A_\v", "*:", "<\ufeff\u0082/|", "a!<", "a\u0096∦\u2000$؈@@@A", "a⨘", "ª\U0001343f***oⅶ฿\x00 !a🨲^?₠#", "̳"***, resource.PropertyValue***V:(*resource.Secret)(0xc000824fd0)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:(*archive.Archive)(0xc00100da40)***, Known:true, Secret:false, Dependencies:[]urn.URN***"", "\ufeffȺ[aᶰǈॎ~\v\ufeff #₶\x01^_˚\x1b\\€ⸯ!aa"***, "\\!":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*archive.Archive)(0xc00110edc0)***, resource.PropertyValue***V:3.0423732918007356***, resource.PropertyValue***V:"¾\r\u0092?"***, resource.PropertyValue***V:(*asset.Asset)(0xc00110ee10)***, resource.PropertyValue***V:false***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"\x1b\u0093&Ⱥ\U000e0042*"***, resource.PropertyValue***V:true***, resource.PropertyValue***V:false***, resource.PropertyValue***V:(*asset.Asset)(0xc00110ee60)***, resource.PropertyValue***V:false***, resource.PropertyValue***V:-4.9897966875095516e-111***, resource.PropertyValue***V:3.091558238780454e-16***, resource.PropertyValue***V:".aA𝈡®˔#↮"***, resource.PropertyValue***V:true***, resource.PropertyValue***V:false***, resource.PropertyValue***V:-4.770032191423094e+243***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:-4.107292791674546e-157***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:false, Dependencies:[]urn.URN***"", "%-^aʺ#Ꮷ_?º₢'", "+︀=", "A`\n\x00", "a"***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:true, Dependencies:[]urn.URN(nil)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:true, Dependencies:[]urn.URN(nil)***, resource.PropertyValue***V:(*asset.Asset)(0xc00110ef50)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:-2.661127870223222e+57***, Known:true, Secret:false, Dependencies:[]urn.URN***"", ".\n", "d", "꙲\\𒐦\rAਃǅ/໒~A\\ʲA\n٠"***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:true, Secret:false, Dependencies:[]urn.URN***"%\u009fȺ₧", "*", "+ga", "@~\"A큿x;", "[🄌?$*aa₭B", "\\#ᛯ\U00013432๓A~a", "̎=᪾", "࿃", "ᾯ-A»Ⱥ", "�𐅄¨𞴙"***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"🯹\U0001343f*"***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:(*asset.Asset)(0xc00110eff0)***, resource.PropertyValue***V:"\ue03a:A.\U0010e576₌ᛯ"***, resource.PropertyValue***V:(*resource.Secret)(0xc0008256f0)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.PropertyMap***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:4.620591249793152e-150***, resource.PropertyValue***V:3.8714829247182934***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:(*archive.Archive)(0xc00110f090)***, resource.PropertyValue***V:-7.607556460655681e+192***, resource.PropertyValue***V:(*asset.Asset)(0xc00110f0e0)***, resource.PropertyValue***V:(*asset.Asset)(0xc00110f130)***, resource.PropertyValue***V:(*archive.Archive)(0xc00110f310)***, resource.PropertyValue***V:true***, resource.PropertyValue***V:(*archive.Archive)(0xc00110f360)***, "`":resource.PropertyValue***V:resource.PropertyMap***, "a\x04?ᾊ|aaᛮ$ᵎ":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*asset.Asset)(0xc00110eaf0)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:-0.2314658504910767***, Known:true, Secret:false, Dependencies:[]urn.URN***"@\x1a𒑮Ⅲ", "̒%", "؋a!ɜ", "⃠৭\r~:"***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:false, Dependencies:[]urn.URN***":", "=", "a!~\u2066a❛?", "~\ue003\"̂0:¢\u2062[@!", "~\U000134361U\u009e፮\"#", "§A[\x00,Ⱥ೦\\", "֏\"", "⃞_\U000e0033𐓩", "↻*^", "𖽞!?"***, resource.PropertyValue***V:resource.PropertyMap***"$\nᾟˠ":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "$a@":resource.PropertyValue***V:interface ***(nil)***, "****~༂ǅ^$̭³²:⃞݇":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "0":resource.PropertyValue***V:(*asset.Asset)(0xc00110eb40)***, "Aa²":resource.PropertyValue***V:"/::\x7f!\ue1f1¸+ᾍ˺A\t୮=A"***, "a":resource.PropertyValue***V:2.8548446917003067e+21***, "º_***Ṃ𑇒\ue007Ⱥ":resource.PropertyValue***V:-8.146637572396831e-151***, "꙲િ𑋡𲎯":resource.PropertyValue***V:-0.026152769586587965***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:"(꣐🡾⃣?A"***, Known:true, Secret:false, Dependencies:[]urn.URN***"𞀸$?\u2009~/\nA\u206a!"***, resource.PropertyValue***V:true***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:interface ***(nil)***, "^(⃣ʺ𞁭𖽦𑿠@+ᾚ_!":resource.PropertyValue***V:(*archive.Archive)(0xc00110ebe0)***, "^\U000e0023௧ע":resource.PropertyValue***V:false***, "a\v#/ªa\ue004-,\u00a0\"??":resource.PropertyValue***V:(*archive.Archive)(0xc00110eb90)***, "a⃝\rȺ?³?":resource.PropertyValue***V:(*asset.Asset)(0xc00110ec30)***, "~\U000fe6c1ᾭꢿAȺ﹩":resource.PropertyValue***V:interface ***(nil)***, "⇿ذ﷼ᛮ,":resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:false, Dependencies:[]urn.URN***"", "?~#࿇ᶇ\x7f", "a#!#፮^?", "|߿﷼_***ܫ", "꜈ו‽"***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:true, Dependencies:[]urn.URN(nil)***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:-0.00890525384549079***, resource.PropertyValue***V:true***, resource.PropertyValue***V:(*asset.Asset)(0xc00110ed20)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:"?.a+ᾏ̂"***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"aࠤ\ufeff¦"***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:(*asset.Asset)(0xc00110ed70)***, "a <?₮ٴ৵=":resource.PropertyValue***V:resource.PropertyMap***"૾A":resource.PropertyValue***V:resource.PropertyMap***, "ᛯ":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:true***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "~ऻ`":resource.PropertyValue***V:resource.PropertyMap***" \u2003∁A":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:"ʽºa"***, "A١̀𒐠aaW�½؛":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:(*asset.Asset)(0xc00100de50)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "D𞲯,| A\u00ad#A@\u0600০\x7f\u202b|+ਾ":resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:"\u009c\U000e0036~꩔\r(⃣\x7f"***, "~ᾘAA?𝨤A?@O+̆":resource.PropertyValue***V:false***, "𐞖Ⱥ$�©ʱॊᾝ":resource.PropertyValue***V:(*asset.Asset)(0xc00100de00)***, "s₥":resource.PropertyValue***V:[]resource.PropertyValue***, "ȺaȺ🏼":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:"'"***, resource.PropertyValue***V:(*archive.Archive)(0xc00100da90)***, resource.PropertyValue***V:(*archive.Archive)(0xc00100dc70)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:7.376724196770143e-69***, resource.PropertyValue***V:(*asset.Asset)(0xc00100dcc0)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:"_"***, resource.PropertyValue***V:(*asset.Asset)(0xc00100dd10)***, resource.PropertyValue***V:"ᾉ"***, resource.PropertyValue***V:(*asset.Asset)(0xc00100dd60)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:3.5108401365578175***, resource.PropertyValue***V:(*archive.Archive)(0xc00100ddb0)***, resource.PropertyValue***V:-1.573815547120799e-17***, "\ufeff":resource.PropertyValue***V:true***, "ǋ_":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:(*resource.Secret)(0xc0008257b0)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:-0.687617509032493***, Known:true, Secret:false, Dependencies:[]urn.URN***"ཿ\ufeff\u0097%˶𝙪ᾎ"***, resource.PropertyValue***V:-0.00467745217247284***, resource.PropertyValue***V:(*resource.Secret)(0xc0008257f0)***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:2.4833494648554686e+40***, Known:true, Secret:false, Dependencies:[]urn.URN***"", " ⊻∳", ",𑨳", ";¸ב$^�#", "?aa￦\t⃠#?~~ೇ\x06ؾ-∛\ue030!", "@~²a𞥃`A", "@꣔Aᮦa~🏾*?✫⾓!ð𑼵'FȺ\u200aຊ", "a\x02_⃤1Aཛʵ ౢǅa�⃣?£౺_a\U000f773f\t# Ⅳ៛AȺ@%^@a_ʺ&AaAA\r", "꙰Aƻ"***, resource.PropertyValue***V:(*asset.Asset)(0xc00110f4a0)***, resource.PropertyValue***V:(*resource.Secret)(0xc000825850)***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:true***, resource.PropertyValue***V:false***, resource.PropertyValue***V:2.9370551448875288e+112***, resource.PropertyValue***V:","***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:1.7976931348623157e+308***, resource.PropertyValue***V:[]resource.PropertyValue***, resource.PropertyValue***V:[]resource.PropertyValue***, resource.PropertyValue***V:resource.PropertyMap***, resource.PropertyValue***V:(*resource.Secret)(0xc000825880)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:"!\""***, Known:true, Secret:false, Dependencies:[]urn.URN***"A₥\u0091.𝼒", "a'̕ǈ.ǲ\a@∠�", "⃟߾_!=~"***, resource.PropertyValue***V:(*resource.Secret)(0xc0008258c0)***, resource.PropertyValue***V:(*resource.Secret)(0xc0008258e0)***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:true***, resource.PropertyValue***V:"\u0084\rh\u008b="***, "Ⱥ᥎":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:true, Dependencies:[]urn.URN(nil)***, "́*\u205fة_a&[a***叓Ⱥ?":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:(*archive.Archive)(0xc00110f3b0)***, "*𞴂":resource.PropertyValue***V:"+ ^΅ʵb<\rc"***, "***":resource.PropertyValue***V:true***, "~@:,a⦽":resource.PropertyValue***V:(*archive.Archive)(0xc00110f400)***, "\u06dd𒐎#***@𑆽\t":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "ો":resource.PropertyValue***V:false***, Known:true, Secret:false, Dependencies:[]urn.URN***"<¦:*\ue00e-?\ue000", "@ᾨA?", "|", "²"***, "҈.d":resource.PropertyValue***V:resource.PropertyMap***",.":resource.PropertyValue***V:(*resource.Secret)(0xc000825320)***, "AaA̎\x7f˗":resource.PropertyValue***V:interface ***(nil)***, "***̄":resource.PropertyValue***V:(*resource.Secret)(0xc000825230)***, "ƻ̄Aᱸ":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:true***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:(*asset.Asset)(0xc00110e5a0)***, resource.PropertyValue***V:(*asset.Asset)(0xc00110e5f0)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:(*archive.Archive)(0xc00110e640)***, "٣~\x7f~ᾞ?,":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:true***, resource.PropertyValue***V:false***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:"A?́a꜕A᪾ǋ\u2029\x1b ̚"***, resource.PropertyValue***V:-0.7618408203125***, resource.PropertyValue***V:-1.3666276782751083***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:interface ***(nil)***, resource.PropertyValue***V:(*asset.Asset)(0xc00110eaa0)***, resource.PropertyValue***V:"\r"***, "\u202b^#ǃ꜊a~ʷ-ʹ(\x1bʰ:༰𐋱+<\ue032&©ی":resource.PropertyValue***V:(*resource.Secret)(0xc0008252f0)***, "𐞰(":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:false***, resource.PropertyValue***V:-1.2675636421045056***, resource.PropertyValue***V:(*archive.Archive)(0xc00110e820)***, resource.PropertyValue***V:2.507642632806827e-93***, resource.PropertyValue***V:"\tŮ+a\tAⅤ*^'"***, resource.PropertyValue***V:(*asset.Asset)(0xc00110e870)***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:(*archive.Archive)(0xc00110ea00)***, resource.PropertyValue***V:-1.7692394107580185***, resource.PropertyValue***V:"िȺA"***, resource.PropertyValue***V:(*archive.Archive)(0xc00110ea50)***, resource.PropertyValue***V:interface ***(nil)***, "\U000f5116҉\ue000":resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:interface ***(nil)***, "\x1b:ĮऻᛮⅫ":resource.PropertyValue***V:(*archive.Archive)(0xc00110e780)***, "(M$\ufeffৈA%৹\ue001\u2005$":resource.PropertyValue***V:(*archive.Archive)(0xc00110e6e0)***, "DȺၖ\u202e+":resource.PropertyValue***V:(*archive.Archive)(0xc00110e7d0)***, "~":resource.PropertyValue***V:"&&₊⍦⋆Ⱥ"***, "\u00ada߾n╙ãⅯ⃞#\U0001bca2":resource.PropertyValue***V:1.3124921145760028e+137***, "ǁ-꜌($+࿂»𒑟`҈=~ᶉ϶$/B鐯aaa^":resource.PropertyValue***V:(*asset.Asset)(0xc00110e730)***, "𐭻\x7fA⎱୶":resource.PropertyValue***V:"\ufeff૱"***, "𝟚!":resource.PropertyValue***V:false***, "\u1680ि":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:false***, resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:"Ⅲ#"***, Known:true, Secret:false, Dependencies:[]urn.URN***"𞀻¯a𞁍𝟛𐞋"***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:true, Dependencies:[]urn.URN(nil)***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:-3.6680729531885305e-19***, "\t!AȺAa":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "\v߁":resource.PropertyValue***V:(*archive.Archive)(0xc00110e0a0)***, " ̧¦A୲A":resource.PropertyValue***V:"٠~;\uef05+౺'"***, "\"\x00𞥃\u00a0":resource.PropertyValue***V:true***, "a\x00 ":resource.PropertyValue***V:(*asset.Asset)(0xc00100dea0)***, "aA𝩔*\ue001@a`(𝡄\x1b$\u2002":resource.PropertyValue***V:(*archive.Archive)(0xc00110e050)***, "|^/":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "|ᾉ":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "~~ʲ*﷼?\\ ӑ;+â\ue001":resource.PropertyValue***V:0.5712894797325134***, "⃤\U0010fffd:_P\na\x7f₲〡@ל*٣":resource.PropertyValue***V:(*asset.Asset)(0xc00110e000)***, "𞲰̗\v՜Caa\x1b;\U000e0049\U000e005e⼨":resource.PropertyValue***V:false***, resource.PropertyValue***V:resource.PropertyMap***"\x05~":resource.PropertyValue***V:"@"***, resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.PropertyMap***"a":resource.PropertyValue***V:(*archive.Archive)(0xc00110e140)***, "\ue002*₡,~A\tA":resource.PropertyValue***V:-5.545604399343058e-32***, resource.PropertyValue***V:(*archive.Archive)(0xc00110e190)***, resource.PropertyValue***V:(*resource.Secret)(0xc0008251f0)***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:"~𑶋\u2008+ൗ˗\U000e003e\x00!\ue007⪚ú"***, resource.PropertyValue***V:(*asset.Asset)(0xc00110e1e0)***, resource.PropertyValue***V:(*archive.Archive)(0xc00110e370)***, resource.PropertyValue***V:(*archive.Archive)(0xc00110e3c0)***, resource.PropertyValue***V:(*archive.Archive)(0xc00110e500)***, resource.PropertyValue***V:(*archive.Archive)(0xc00110e550)***, "⃞&ᾜ\u200d@":resource.PropertyValue***V:-2.6465543033887956e-142***, "⯡=Ѵ\ue003 £\ue001쑟aᾯ#;g\U000e007f¤ꯤ/\U000f5ba8!⟝Ⅱ\x02\v":resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, "\U000ff096\x00":resource.PropertyValue***V:-3.0742958188056946e-06***, resource.PropertyValue***V:resource.PropertyMap***"\x01~":resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:(*resource.Secret)(0xc000825c40)***, resource.PropertyValue***V:(*resource.Secret)(0xc000825c60)***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:"$Tᛯ˝a~a"***, Known:true, Secret:true, Dependencies:[]urn.URN***"\x1b³㉞_"***, resource.PropertyValue***V:"A^Áظ(A~A"***, resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:[]resource.PropertyValue***resource.PropertyValue***V:""***, resource.PropertyValue***V:(*archive.Archive)(0xc00110fcc0)***, resource.PropertyValue***V:true***, resource.PropertyValue***V:false***, resource.PropertyValue***V:""***, resource.PropertyValue***V:false***, resource.PropertyValue***V:"?ǈ𒑅;'𝚫&\"৵\v"***, resource.PropertyValue***V:-2.4164322989973353e+288***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:[]resource.PropertyValue***, resource.PropertyValue***V:resource.PropertyMap***"\r|п൏\ue006ǋ":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:(*archive.Archive)(0xc00110fd10)***, Known:true, Secret:false, Dependencies:[]urn.URN***"﮼ᾮ", "﮽𞲰"***, "A*":resource.PropertyValue***V:resource.PropertyMap***"<^":resource.PropertyValue***V:"؎`/?˂=\u2028^#\u00adAh�A"***, "?<@a-\u00a0":resource.PropertyValue***V:(*archive.Archive)(0xc00110fd60)***, "́;\\₷":resource.PropertyValue***V:"+́ܞ\U000e0027"***, "\x7f,=ËAa[ः᪾~@=𐴵.\v⃠\x7f?=A|୰𝓛a":resource.PropertyValue***V:(*resource.Secret)(0xc000825cf0)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:5.1097158028641766e+128***, Known:true, Secret:true, Dependencies:[]urn.URN***"", "\x01󠆓፴²\U001081c7༣", "'ʴ^=A³<\ue000?\U000f8424:ˈ(꜁#฿", "+<ᾎȺ%/", ":𒑮\x7f_", "A", "\\¤\u00a0+\U000f0628~\ue000𝗠 \u00ad#", "ˁ🯰४", "୴৵҈𑇀\u202e𒐬a⯏�⅄ऻǀ'", "𣍡ःaⅤ^Ⱥ\x018"***, resource.PropertyValue***V:true***, resource.PropertyValue***V:(*resource.Secret)(0xc000825d00)***, resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:(*asset.Asset)(0xc00110fdb0)***, Known:true, Secret:true, Dependencies:[]urn.URN***"", "\x00??҉?𞤡᳡", ":੶", "̅A"***, resource.PropertyValue***V:resource.Computed***Element:resource.PropertyValue***V:""***, resource.PropertyValue***V:resource.PropertyMap***"":resource.PropertyValue***V:resource.PropertyMap***"%":resource.PropertyValue***V:interface ***(nil)***, "?ab<a":resource.PropertyValue***V:(*asset.Asset)(0xc00113a000)***, "̀?\x7f":resource.PropertyValue***V:interface ***(nil)***, "A":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:(*archive.Archive)(0xc00110fea0)***, Known:true, Secret:false, Dependencies:[]urn.URN***".¤"***, "a":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:"𝞸𞥃"***, Known:true, Secret:false, Dependencies:[]urn.URN***"௸/A¥[aᾝ৹"***, "a\u2003":resource.PropertyValue***V:3.8392238435728152e-239***, "ǈ҉a#⨲ʱA\u2028ꟅA\u2068@[$࠰":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:interface ***(nil)***, Known:false, Secret:false, Dependencies:[]urn.URN***"$_", "%!ᛮ/፻\ue17d", "@:a\x7fᩰƻ", "|=", "±⃢ᾯ/", "\u0602a", "ᾘ,₸+.ª῞"***, "ʱ𝜬+＇ᾉ":resource.PropertyValue***V:"&Aȥ\ue003"***, "҈ࣉ`<Ტaǋ":resource.PropertyValue***V:resource.PropertyMap***, "₠˘":resource.PropertyValue***V:resource.Output***Element:resource.PropertyValue***V:1.119140625***, Known:true, Secret:false, Dependencies:[]urn.URN***"\";𒐫%ᾍ$~𝓀\ue000(\ue007\v꭛A", "&ᛯ!_", "©$A\bA¾:Ꭻ\u1680", "\u200c$º:\\\"a_𑿞𞻱<a&\U000fe04ea;ऻ৲\ue008ᾉ\u202e𝪊@\r\ue001Aa\x01", "🀢#"***, "\ue000A\ue028/'Ｅᛮ<ܠ:˕~A":resource.PropertyValue***V:[]resource.PropertyValue***resource.Property<... truncated>

Diff:
--- Expected
+++ Actual
@@ -1628,22 +1628,3 @@
***,
-   Array: (*resource.ArrayDiff)(***
-    Adds: (map[int]resource.PropertyValue) (len=2) ***
-     (int) 0: (resource.PropertyValue) ***
-      V: (bool) false
-     ***,
-     (int) 1: (resource.PropertyValue) ***
-      V: ([]resource.PropertyValue) (len=1) ***
-       (resource.PropertyValue) ***
-        V: (string) (len=9) "⭄|\v\u008b҉"
-       ***
-      ***
-     ***
-    ***,
-    Deletes: (map[int]resource.PropertyValue) ***
-    ***,
-    Sames: (map[int]resource.PropertyValue) ***
-    ***,
-    Updates: (map[int]resource.ValueDiff) ***
-    ***
-   ***),
+   Array: (*resource.ArrayDiff)(<nil>),
Object: (*resource.ObjectDiff)(<nil>)
Test:       	TestDiffCompatibility

Traceback (most recent call last):
File "/home/runner/work/pulumi/pulumi/scripts/go-test.py", line 133, in <module>
raise e
File "/home/runner/work/pulumi/pulumi/scripts/go-test.py", line 106, in <module>
sp.check_call(args, shell=False)
~~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
File "/opt/hostedtoolcache/Python/3.14.6/x64/lib/python3.14/subprocess.py", line 420, in check_call
raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '['gotestsum', '--junitfile', '/home/runner/work/pulumi/pulumi/junit/92270e5f-78ff-40fc-9370-88efb088be50-junit.xml', '--jsonfile', '/home/runner/work/pulumi/pulumi/test-results/92270e5f-78ff-40fc-9370-88efb088be50.json', '--packages', 'github.com/pulumi/pulumi/sdk/v3/go/auto/automation/optorgsearch github.com/pulumi/pulumi/sdk/v3/go/common/util/tracing github.com/pulumi/pulumi/sdk/v3/go/pulumi-internal github.com/pulumi/pulumi/sdk/v3/go/common/providers github.com/pulumi/pulumi/sdk/v3/go/auto/opthistory github.com/pulumi/pulumi/sdk/v3 github.com/pulumi/pulumi/sdk/v3/go/common/util/result github.com/pulumi/pulumi/sdk/v3/go/property github.com/pulumi/pulumi/sdk/v3/go/common/encoding github.com/pulumi/pulumi/sdk/v3/go/common/esc/syntax/encoding github.com/pulumi/pulumi/sdk/v3/go/common/resource/plugin github.com/pulumi/pulumi/sdk/v3/go/internal github.com/pulumi/pulumi/sdk/v3/go/pulumi-internal/rpcdebug github.com/pulumi/pulumi/sdk/v3/go/auto/events github.com/pulumi/pulumi/sdk/v3/go/common/esc/syntax github.com/pulumi/pulumi/sdk/v3/go/pulumi/errors github.com/pulumi/pulumi/sdk/v3/go/common/util/rpcutil/rpcerror github.com/pulumi/pulumi/sdk/v3/go/common/tail github.com/pulumi/pulumi/sdk/v3/go/internal/gen-pux-applyn github.com/pulumi/pulumi/sdk/v3/go/common/testing/diagtest github.com/pulumi/pulumi/sdk/v3/go/common/util/mapper github.com/pulumi/pulumi/sdk/v3/go/common/util/executable github.com/pulumi/pulumi/sdk/v3/go/auto/optup github.com/pulumi/pulumi/sdk/v3/go/common/env github.com/pulumi/pulumi/sdk/v3/go/common/tokens github.com/pulumi/pulumi/sdk/v3/go/common/util/deepcopy github.com/pulumi/pulumi/sdk/v3/go/auto/automation/optcancel github.com/pulumi/pulumi/sdk/v3/go/pulumi/internals github.com/pulumi/pulumi/sdk/v3/go/common/resource/asset github.com/pulumi/pulumi/sdk/v3/go/common/util/contract github.com/pulumi/pulumi/sdk/v3/go/auto/optrefresh github.com/pulumi/pulumi/sdk/v3/go/common/esc/ast github.com/pulumi/pulumi/sdk/v3/go/pulumix github.com/pulumi/pulumi/sdk/v3/go/common/esc/internal/spell github.com/pulumi/pulumi/sdk/v3/go/common/util/logging github.com/pulumi/pulumi/sdk/v3/go/common/version github.com/pulumi/pulumi/sdk/v3/go/common/resource/sig github.com/pulumi/pulumi/sdk/v3/go/auto/optrename github.com/pulumi/pulumi/sdk/v3/go/tools/automation/boilerplate/testing github.com/pulumi/pulumi/sdk/v3/proto/go/testing github.com/pulumi/pulumi/sdk/v3/go/common/esc/diags github.com/pulumi/pulumi/sdk/v3/go/pulumi-internal/gsync github.com/pulumi/pulumi/sdk/v3/go/common/lazy github.com/pulumi/pulumi/sdk/v3/go/common/util/rpcutil github.com/pulumi/pulumi/sdk/v3/go/common/esc github.com/pulumi/pulumi/sdk/v3/go/common/util/httputil github.com/pulumi/pulumi/sdk/v3/go/common/util/ciutil github.com/pulumi/pulumi/sdk/v3/go/common/util/otelreceiver/logging github.com/pulumi/pulumi/sdk/v3/go/common/util/otelreceiver github.com/pulumi/pulumi/sdk/v3/go/auto/optremoterefresh github.com/pulumi/pulumi/sdk/v3/go/common/util/archive github.com/pulumi/pulumi/sdk/v3/go/auto/optimport github.com/pulumi/pulumi/sdk/v3/go/auto/optremotepreview github.com/pulumi/pulumi/sdk/v3/go/common/diag github.com/pulumi/pulumi/sdk/v3/go/auto/automation/optorgsearchai github.com/pulumi/pulumi/sdk/v3/go/common/apitype github.com/pulumi/pulumi/sdk/v3/go/common/util/cmdutil github.com/pulumi/pulumi/sdk/v3/go/common/workspace github.com/pulumi/pulumi/sdk/v3/go/pulumi/policyx github.com/pulumi/pulumi/sdk/v3/go/tools/automation/boilerplate/standard github.com/pulumi/pulumi/sdk/v3/go/common/util/env github.com/pulumi/pulumi/sdk/v3/go/pulumi github.com/pulumi/pulumi/sdk/v3/go/auto/automation/optorgsetdefault github.com/pulumi/pulumi/sdk/v3/go/common/util/gitutil github.com/pulumi/pulumi/sdk/v3/go/common/util/agentdetect github.com/pulumi/pulumi/sdk/v3/go/common/esc/schema github.com/pulumi/pulumi/sdk/v3/go/tools/automation github.com/pulumi/pulumi/sdk/v3/python/toolchain github.com/pulumi/pulumi/sdk/v3/go/auto/automation/optnew github.com/pulumi/pulumi/sdk/v3/go/auto/optremotedestroy github.com/pulumi/pulumi/sdk/v3/go/auto/automation/optimport github.com/pulumi/pulumi/sdk/v3/go/pulumi-internal/netutil github.com/pulumi/pulumi/sdk/v3/go/common/testing github.com/pulumi/pulumi/sdk/v3/go/auto/automation/optorg github.com/pulumi/pulumi/sdk/v3/go/property/testing github.com/pulumi/pulumi/sdk/v3/go/auto/automation/optorggetdefault github.com/pulumi/pulumi/sdk/v3/go/common/resource/urn github.com/pulumi/pulumi/sdk/v3/go/auto/optpreview github.com/pulumi/pulumi/sdk/v3/proto/go github.com/pulumi/pulumi/sdk/v3/go/auto/optlist github.com/pulumi/pulumi/sdk/v3/go/common/resource github.com/pulumi/pulumi/sdk/v3/go/common/promise github.com/pulumi/pulumi/sdk/v3/go/auto/automation/base github.com/pulumi/pulumi/sdk/v3/go/common/util/maputil github.com/pulumi/pulumi/sdk/v3/go/auto/optremoteup github.com/pulumi/pulumi/sdk/v3/go/common/esc/internal/util github.com/pulumi/pulumi/sdk/v3/go/common/util/fsutil github.com/pulumi/pulumi/sdk/v3/go/common/testing/iotest github.com/pulumi/pulumi/sdk/v3/go/auto/automation github.com/pulumi/pulumi/sdk/v3/go/common/util/hgutil github.com/pulumi/pulumi/sdk/v3/go/auto/automation/opt github.com/pulumi/pulumi/sdk/v3/go/common/util/yamlutil github.com/pulumi/pulumi/sdk/v3/go/pulumi/config github.com/pulumi/pulumi/sdk/v3/go/common/diag/colors github.com/pulumi/pulumi/sdk/v3/go/common/slice github.com/pulumi/pulumi/sdk/v3/go/common/resource/config github.com/pulumi/pulumi/sdk/v3/go/pulumi/provider github.com/pulumi/pulumi/sdk/v3/go/pulumi/generate github.com/pulumi/pulumi/sdk/v3/go/common/esc/eval github.com/pulumi/pulumi/sdk/v3/proto/go/codegen github.com/pulumi/pulumi/sdk/v3/go/common/util/errutil github.com/pulumi/pulumi/sdk/v3/go/common/resource/archive', '--', '-count=1', '-cover', '-tags=all', '-timeout', '1h', '-parallel=8', '-shuffle=off', '-p=2', '-race=true', '-coverprofile=/home/runner/work/pulumi/pulumi/coverage/go-test-a678c0e3.cov', '-coverpkg=github.com/pulumi/pulumi/pkg/v3/...,github.com/pulumi/pulumi/sdk/v3/...,github.com/pulumi/pulumi/sdk/go/pulumi-language-go/v3/...,github.com/pulumi/pulumi/sdk/nodejs/cmd/pulumi-language-nodejs/v3/...,github.com/pulumi/pulumi/sdk/python/cmd/pulumi-language-python/v3/...,github.com/pulumi/pulumi/sdk/pcl/v3/cmd/pulumi-language-pcl/...']' returned non-zero exit status 1.
make: *** [Makefile:279: gotestsum/sdk] Error 1
```

--- a/sdk/go/common/resource/property_compatibility_test.go
+++ b/sdk/go/common/resource/property_compatibility_test.go
@@ -122,3 +122,40 @@ func TestDiffCompatibility(t *testing.T) {
 			resource.ToResourcePropertyMap(a).Diff(resource.ToResourcePropertyMap(b)))
 	})
 }
+
+func TestDiffCompatibilityWrappedValues(t *testing.T) {
+	t.Parallel()
+
+	tests := []struct {
+		name string
+		a, b property.Map
+	}{
+		{
+			"secret arrays",
+			property.NewMap(map[string]property.Value{
+				"k": property.New([]property.Value{property.New(1.0)}).WithSecret(true),
+			}),
+			property.NewMap(map[string]property.Value{
+				"k": property.New([]property.Value{property.New(2.0)}).WithSecret(true),
+			}),
+		},
+		{
+			"secret maps with null values",
+			property.NewMap(map[string]property.Value{
+				"k": property.New(map[string]property.Value{"n": property.New(property.Null)}).WithSecret(true),
+			}),
+			property.NewMap(map[string]property.Value{
+				"k": property.New(property.Map{}).WithSecret(true),
+			}),
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			t.Parallel()
+			assert.Equal(t,
+				resource.ToResourceObjectDiff(tt.a.Diff(tt.b)),
+				resource.ToResourcePropertyMap(tt.a).Diff(resource.ToResourcePropertyMap(tt.b)))
+		})
+	}
+}

--- a/sdk/go/property/diff.go
+++ b/sdk/go/property/diff.go
@@ -265,6 +265,8 @@ func (v Value) diff(other Value, opts diffOptions, path Path) *ValueDiff {
 		return &ValueDiff{Old: v, New: other}
 	}
 
+	opaque := v.Secret() || len(v.Dependencies()) > 0
+
 	if v.IsArray() && other.IsArray() {
 		old := v.AsArray()
 		new := other.AsArray()
@@ -293,6 +295,9 @@ func (v Value) diff(other Value, opts diffOptions, path Path) *ValueDiff {
 		if len(adds) == 0 && len(deletes) == 0 && len(updates) == 0 {
 			return nil
 		}
+		if opaque {
+			return &ValueDiff{Old: v, New: other}
+		}
 		return &ValueDiff{
 			Old: v,
 			New: other,
@@ -308,6 +313,9 @@ func (v Value) diff(other Value, opts diffOptions, path Path) *ValueDiff {
 		old := v.AsMap()
 		new := other.AsMap()
 		if diff := old.diff(new, opts, path); diff != nil {
+			if opaque {
+				return &ValueDiff{Old: v, New: other}
+			}
 			return &ValueDiff{
 				Old:    v,
 				New:    other,

